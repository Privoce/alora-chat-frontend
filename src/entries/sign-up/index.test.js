import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {
	SignUpEntry
} from 'entries';

describe('<SignUpEntry />', () => {
	const modelI = {};

	const getWrapper = (model) => {
		return shallow((
			<SignUpEntry {...model} />
		));
	};

	it('renders correctly', () => {
		const wrapper = getWrapper(modelI);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
