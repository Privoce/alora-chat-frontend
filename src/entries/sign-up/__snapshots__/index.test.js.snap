// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignUpEntry /> renders correctly 1`] = `
<AuthContainer
  footerInfo={
    <LabelComponent
      text="Already have an account?"
    />
  }
  footerInfoLink={
    <ButtonComponent
      link={true}
      onClick={[Function]}
      text="Sign In"
      width={80}
    />
  }
  formContainer={<Connect(SignUpFormContainer) />}
  title="Sign Up"
/>
`;
